signature/jws/jws_test.go

```diff
@@ -0,0 +1,23 @@
+package jws
+
+import (
+	"math"
+	"testing"
+)
+
+func Test_convertToMapError(t *testing.T) {
```

Comment:
Can we add an ok case? Eg: Test_convertToMapOk, validate your map is the expected one.

Reply:
Added.